sdks/python/apache_beam/io/gcp/bigquery_write_it_test.py

```diff
@@ -412,11 +420,103 @@ def test_big_query_write_temp_table_append_schema_update(self):
           | 'write' >> beam.io.WriteToBigQuery(
               table_id,
               schema=table_schema,
-              write_disposition=beam.io.BigQueryDisposition.WRITE_APPEND,
+              write_disposition=BigQueryDisposition.WRITE_APPEND,
```

Comment:
style nit: Rest of the file is using `beam.io.BigQueryDisposition` rather than `BigQueryDisposition`.